userguide/sound_classifier/README.md

```diff
@@ -0,0 +1,55 @@
+# Sound Classifier
+
+Given a sound, the goal of the Sound Classifier is to assign it to
+one of a pre-determined number of labels, such as baby crying, siren,
+or dog barking. This Sound Classifier is not intended to be used for
+speech recoginition.
+
+#### Loading Data
+
+For example purposees, we will use the ESC-10 dataset. This dataset
+contain ten classes. Each class has 40 examples with five seconds of
+audio per example.
+
+**Please note:** the ESC-10 dataset is part of a larger [ESC-50 dataset](https://github.com/karoldvl/ESC-50)
+dataset. In order to use the ESC-10, you will need to download the
+[ESC-50 dataset](https://github.com/karoldvl/ESC-50). These two datasets have different licenses.
+Depending on your use case you may be able to the larger ESC-50
```

Comment:
... you may be able to *use* the larger ...

Reply:
Done.